pkg/obsoletetests/ocp_obsoletetests.go

```diff
@@ -0,0 +1,27 @@
+package obsoletetests
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	v1 "github.com/openshift-eng/ci-test-mapping/pkg/api/types/v1"
+)
+
+type OCPObsoleteTestManager struct{}
+
+type obsoleteTestIdentifier struct {
```

Comment:
shall we add an enum for reason and a panic for anything in the set without it so people are on record as to why?